src/mavlink-router/autolog.cpp

```diff
@@ -80,3 +80,12 @@ bool AutoLog::start()
 
     return true;
 }
+
+void AutoLog::print_statistics()
```

Comment:
oh... I think this shows that the autolog functionality should not really be done the way it is right now.... it can lead to more places using the wrong "wrapper object" rather than the real one, particularly on methods implemented on the parent's class like this one.